scripts/sample_size_run.py

```diff
@@ -0,0 +1,34 @@
+def main() -> None:
+    """
+    Calculate sample size based on user inputs for
+        1. metric type: Boolean, Numeric, or Ratio (case insensitive)
+        2. power analysis parameters: alpha, power, and mde
+        3. inputs specific to metric type:
+            * Boolean: probability
+            * Numeric: variance
+            * Ratio: mean and variance of numerator and denominator and their covariance
+
+    """
+    from sample_size.sample_size_calculator import SampleSizeCalculator
+    from scripts.utils import get_alpha
+    from scripts.utils import get_variable_from_input
+    from scripts.utils import register_metric
+
+    # Get alpha for power analysis
+    alpha = get_alpha()
+    if alpha:
+        calculator = SampleSizeCalculator(alpha)
```

Comment:
does that mean `power` is fixed from the script that we don't want users to input it?

Reply:
Internally we supports configurable alpha for now, so the script to configure power is not included here. IMO, most of the users will not update statistical power, asking them to input the power may add inconvenience.

Reply:
note added